tests/build.gradle

```diff
@@ -22,7 +22,12 @@ sourceSets {
 }
 
 dependencies {
-    implementation project(path: ':neoforge', configuration: 'runtimeElements')
+    implementation(neoforgeProject) {
```

Comment:
This logic is incorrect. It will select the _runtime_ elements of the neoforge project, when selecting stuff for the _compile_ classpath. Instead, I would set that attribute on the `compileClasspath` (and equivalent junit) configurations)